staging/src/k8s.io/client-go/pkg/third_party/forked/golang/json/fields.go

```diff
@@ -1,501 +0,0 @@
-// Copyright 2013 The Go Authors. All rights reserved.
```

Comment:
ah, was in the "move workqueue to client-go " commit, which I expected to just be renames and import changes